{"uuid": "2eb3f173-ba3a-4620-8f76-9abc8e8ad563", "children": ["829e64a7-0827-4aa0-8190-fea0e627df0e", "61cf650a-4971-4ba3-985d-21e9c587c750", "49776485-efd1-45bf-8cf3-cc436c02f3cb", "1d873f46-4c21-477b-a2af-362f85af1a90"], "befores": [{"name": "get_token", "status": "broken", "statusDetails": {"message": "AttributeError: 'Logger' object has no attribute 'logger'\n", "trace": "  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/tim/Desktop/test_pro/pyframe/testcases/conftest.py\", line 32, in get_token\n    log.logger.info(\"Authorization:Bearer \"+token[0])\n    ^^^^^^^^^^\n"}, "start": 1716099598805, "stop": 1716099599392}], "start": 1716099598805, "stop": 1716099599485}